### ISSUE TITLE
add tests in client/services/code_evaluators/

### ISSUE THREAD

**Issue by lumig242** [google/tie#224]
- [x] CodeRunnerDispatcherService.js (one line uncovered due to failure to use Sk library)

- [ ] PythonCodeRunnerService.js (blocked by failure to use Sk library)

**Comment by lumig242** [comment]
pr for `CodeRunnerDispatcherService`: #220

**Comment by seanlip** [comment]
Superseded by #252, closing. Thanks @lumig242!
